packages/frontend-2/components/server-management/UserRoleSelect.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <FormSelectBase
+    v-model="selectedValue"
+    :items="roles"
+    :label="'Role'"
+    :name="'userRole'"
```

Comment:
```suggestion
    label="Role"
    name="userRole"
```

Reply:
https://laracasts.com/discuss/channels/vue/when-to-use-colon-when-passing-data-to-props?page=1&replyId=294073